toolkit/cli/smart-contracts-commands/src/governance.rs

```diff
@@ -58,9 +59,12 @@ impl InitGovernanceCmd {
 pub struct UpdateGovernanceCmd {
 	#[clap(flatten)]
 	common_arguments: crate::CommonArguments,
-	/// Governance authority hash to be set.
-	#[arg(long, short = 'g')]
-	new_governance_authority: MainchainKeyHash,
+	/// New governance authorities keys hashes, hex encoded, space delimited, order does not matter
+	#[arg(short = 'g', long, num_args = 1.., value_delimiter = ' ')]
+	new_governance_authority: Vec<MainchainKeyHash>,
+	/// Governance threshold to be set
+	#[arg(long)]
+	new_governance_threshold: u8,
```

Comment:
Can we return error immediately when threshold is greater than new governance authorities length?

It won't save the day if key hashes are invalid (users don't have keys for them)